src/Script/navBar.js

```diff
@@ -1,14 +1,23 @@
 $(document).ready(() => darkModeToggleValidator());
 
-let navBarActive = document.getElementById("navBarActive").innerHTML;
-
-(navBarActive == "homePage") ? (document.getElementById("homeNav").classList.add("active"))
-    : (navBarActive == "projectPage" || navBarActive == "ticketPage") ? (document.getElementById("projectNav").classList.add("active"))
-        : (navBarActive == "registerPage" || navBarActive == "loginPage" || navBarActive == "adminPage" || navBarActive == "managerPage") ? (document.getElementById("accountNav").classList.add("active"))
-            : null;
-
 $('#searchBarInput').on("keypress", (e) => { if (e.keyCode == 13) searchBar(); });
 
+class Navbar
+{
+     static homeActiveTab()
+    {
+        document.getElementById("homeNav").classList.add("active");
```

Comment:
Use same line curly start brackets as it is JS style to do so 
e.g.
static homeActiveTab() {
// blah blah 
}